### ISSUE TITLE
Error: Empty Response from server. Do you want to review mount point config in admin settings page

### ISSUE THREAD

**Issue by djvanny** [hevelius/files_external_onedrive#29]
Hello,
thanks for this app. Unfortunately i can connect to my ondrive once only. The second login f.x. after leaving nextcloud fails. No Error Logging. This is the only exeption i got:

Error: Empty Response from server. Do you want to review mount point config in admin settings page

yes /no

**Comment by clmnsr** [comment]
I have the same problem :-(

**Comment by uwedisch** [comment]
I have the same problem, first access won't work, following that will work.

**Comment by djvanny** [comment]
hello uwedisch, 
thanks for your reply, I'm not sure what kind of thing i'm seeing there. Is this a bugfix via terminal?

**Comment by uwedisch** [comment]
No, not a bug fix. A mutual confirmation that it's also with console the same problem.

**Comment by Nicicalu** [comment]
Same here, I will only work the first time 👎

**Comment by martinlentink** [comment]
Unfortunately I have the exact same issue: the first time it works, but after I leave the mounted share, I cannot get back in. If there's anything you want me to try, please let me know!

**Comment by hevelius** [comment]
Hi all, any kind of error releated on Background Jobs in NC logs? The auto-renew is managed by a background jobs.

**Comment by martinlentink** [comment]
I will have a look as soon as I can. Right now I'm trying to find a fix for a nextcloud updater issue..
